python/dgllife/utils/splitters.py

```diff
@@ -574,7 +577,8 @@ def train_val_test_split(dataset, mols=None, sanitize=True, include_chirality=Fa
                     val_indices.extend(group_indices)
             else:
                 train_indices.extend(group_indices)
-
+        assert len(set(train_indices).intersection(set(val_indices))) == 0, "Failed to successfully split using scaffolds"
+        assert len(set(test_indices).intersection(set(val_indices))) == 0, "Failed to successfully split using scaffolds"
```

Comment:
Why did you add the two assertions here? L572-L579 should exclude the possibility of overlappings.

Reply:
I had these for debugging. I'll delete them and push the change. Thanks!!